client/src/test/java/io/pravega/client/netty/impl/SessionHandlerTest.java

```diff
@@ -0,0 +1,225 @@
+/**
```

Comment:
Roll back changes to the license header, please.

Reply:
corrected it.